.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 100
```

Comment:
`end_of_line` as well

Reply:
Setting the different `max_line_length` to .editorconfig and .prettierrc is my mistake, I'll set these as 100 for both files instead of removing it from .editorconfig.

It looks redundant indeed, but the coverage of supporting IDE for .editorconfig is wider than the prettier plugin, I'd like it to remain.